lib/hooks/backlog/hook.rb

```diff
@@ -0,0 +1,23 @@
+module Idobata::Hook
+  class Backlog < Base
+    screen_name 'backlog'
+    icon_url hook_image_url('icon.png')
+    template_name { "#{type}.html.haml" }
+
+    helper Helper
+
+    private
+
+    def type
+      case payload.type
+      when 1, 2, 3, 4, 14, 17
+        'issue'
+      end
```

Comment:
Also, would you please skip processing when receiving unsupported event? Like [this](https://github.com/idobata/idobata-hooks/blob/6f870a6e243ea3c4a2ca6404f3062b5f6d521f1a/lib/hooks/bitbucket/hook.rb#L47).